### PR TITLE
[#1310] Release 2-0-3

### LINKED ISSUE #1310: Release 2.0.3

### PR DESCRIPTION
**Pull request by LukasKorba**
Closes #1310 

- checkpoints updated
- FFI version bumped
- other dependencies bumped as well

This code review checklist is intended to serve as a starting point for the author and reviewer, although it may not be appropriate for all types of changes (e.g. fixing a spelling typo in documentation).  For more in-depth discussion of how we think about code review, please see [Code Review Guidelines](../blob/main/CODE_REVIEW_GUIDELINES.md).

# Author

- [x] Self-review: Did you review your own code in GitHub's web interface? Code often looks different when reviewing the diff in a browser, making it easier to spot potential bugs.
- [ ] Automated tests: Did you add appropriate automated tests for any code changes?
- [x] Code coverage: Did you check the code coverage report for the automated tests?  While we are not looking for perfect coverage, the tool can point out potential cases that have been missed.
- [ ] Documentation: Did you update Docs as appropiate? (E.g [README.md](../blob/main/README.md), etc.)
- [x] Run the app: Did you run the app and try the changes? 
- [ ] Did you provide Screenshots of what the App looks like before and after your changes as part of the description of this PR? (only applicable to UI Changes)
- [x] Rebase and squash: Did you pull in the latest changes from the main branch and squash your commits before assigning a reviewer? Having your code up to date and squashed will make it easier for others to review. Use best judgement when squashing commits, as some changes (such as refactoring) might be easier to review as a separate commit.

# Reviewer

- [ ] Checklist review: Did you go through the code with the [Code Review Guidelines](../blob/main/CODE_REVIEW_GUIDELINES.md) checklist?
- [ ] Ad hoc review: Did you perform an ad hoc review?  _In addition to a first pass using the code review guidelines, do a second pass using your best judgement and experience which may identify additional questions or comments. Research shows that code review is most effective when done in multiple passes, where reviewers look for different things through each pass._
- [ ] Automated tests: Did you review the automated tests?
- [ ] Manual tests: Did you review the manual tests?_You will find manual testing guidelines under our [manual testing section](../blob/mater/docs/testing/manual_testing)_
- [ ] How is Code Coverage affected by this PR? _We encourage you to compare coverage befor and after your changes and when possible, leave it in a better place. [Learn More...](../blob/master/docs/testing/local_coverage.md)_
- [ ] Documentation: Did you review Docs, [README.md](../blob/master/README.md), [LICENSE.md](../blob/master/LICENSE.md), and [Architecture.md](../blob/master/docs/Architecture.md) as appropriate?
- [ ] Run the app: Did you run the app and try the changes? While the CI server runs the app to look for build failures or crashes, humans running the app are more likely to notice unexpected log messages, UI inconsistencies, or bad output data.

### CODE REVIEW

**Review by HonzaR — APPROVED** [review]

File: Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2250000.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "network": "main",
+  "height": "2250000",
+  "hash": "00000000012b3fe412a00b795a282135c30abd4b46d2bf2e7d0168e5c947ebf5",
```
Comment (daira) [inline Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2250000.json:4]: This hash matches the expected height on https://zcashblockexplorer.com/blocks/00000000012b3fe412a00b795a282135c30abd4b46d2bf2e7d0168e5c947ebf5 . I have not checked the other fields.

File: Example/ZcashLightClientSample/ZcashLightClientSample.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 2]
```diff
@@ -158,8 +158,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/zcash-hackworks/zcash-light-client-ffi",
       "state" : {
-        "revision" : "9bc5877ef6302e877922f79ebead52e50bce94fd",
-        "version" : "0.4.0"
+        "revision" : "514dcd7e6fbfa252bf36d9f00d6b98f465a70704",
```
Comment (daira) [inline Example/ZcashLightClientSample/ZcashLightClientSample.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:161]: I checked this hash.

File: Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2252500.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "network": "main",
+  "height": "2252500",
+  "hash": "000000000098a1bb57555b406c0ec07f380785e7edeee2819739633180d73f1b",
```
Comment (daira) [inline Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2252500.json:4]: This hash matches the expected height on https://zcashblockexplorer.com/blocks/000000000098a1bb57555b406c0ec07f380785e7edeee2819739633180d73f1b .

File: Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2255000.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "network": "main",
+  "height": "2255000",
+  "hash": "0000000000b346a9a52c1d16e0f90233c393f21086329eba8e6dc54656f6649b",
```
Comment (daira) [inline Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2255000.json:4]: This hash matches the expected height on https://zcashblockexplorer.com/blocks/0000000000b346a9a52c1d16e0f90233c393f21086329eba8e6dc54656f6649b .

File: Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2257500.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
+{
+  "network": "main",
+  "height": "2257500",
+  "hash": "00000000011681f1de9045b5e9f19725f14f5e0dbf7690b481cfec7830d34076",
```
Comment (daira) [inline Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2257500.json:4]: This hash matches the expected height on https://zcashblockexplorer.com/blocks/00000000011681f1de9045b5e9f19725f14f5e0dbf7690b481cfec7830d34076 .

File: Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2262500.json [hunk 6]
```diff
@@ -0,0 +1,8 @@
+{
+  "network": "main",
+  "height": "2262500",
+  "hash": "0000000000a68976414388707477790c7c88113a618cb88566b83099e0fbf75a",
```
Comment (daira) [inline Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2262500.json:4]: This hash matches the expected height on https://zcashblockexplorer.com/blocks/0000000000a68976414388707477790c7c88113a618cb88566b83099e0fbf75a .

File: Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2265000.json [hunk 7]
```diff
@@ -0,0 +1,8 @@
+{
+  "network": "main",
+  "height": "2265000",
+  "hash": "00000000014e82151bd0bd894f000dd8d140de2f225b4560d8b64f4016c6a623",
```
Comment (daira) [inline Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2265000.json:4]: This hash matches the expected height on https://zcashblockexplorer.com/blocks/00000000014e82151bd0bd894f000dd8d140de2f225b4560d8b64f4016c6a623 .

File: Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2267500.json [hunk 8]
```diff
@@ -0,0 +1,8 @@
+{
+  "network": "main",
+  "height": "2267500",
+  "hash": "00000000013551a841d3fdeb3c4575e88cd86d0afe0c83ecc6039216e5b0163c",
```
Comment (daira) [inline Sources/ZcashLightClientKit/Resources/checkpoints/mainnet/2267500.json:4]: This hash matches the expected height on https://zcashblockexplorer.com/blocks/00000000013551a841d3fdeb3c4575e88cd86d0afe0c83ecc6039216e5b0163c .

File: Sources/ZcashLightClientKit/Resources/checkpoints/testnet/2540000.json [hunk 9]
```diff
@@ -0,0 +1,8 @@
+{
+  "network": "test",
+  "height": "2540000",
+  "hash": "0010c1345d4412a8367aec519bee8d566053cab59109c333201ce35eefe9dd21",
```
Comment (daira) [inline Sources/ZcashLightClientKit/Resources/checkpoints/testnet/2540000.json:4]: I have not checked the hashes for testnet checkpoints.

**Comment by tw0po1nt** [comment]
@LukasKorba Can you tag this release?

**Pull request merged**